android/src/main/java/com/theoplayer/presentation/PresentationManager.kt

```diff
@@ -279,10 +300,41 @@ class PresentationManager(
   }
 }
 
-inline fun <reified T : View> getClosestParentOfType(view: View?): T? {
-  var parent: ViewParent? = view?.parent
-  while (parent != null && parent !is T) {
-    parent = parent.parent
+inline fun <reified T : View> ViewGroup.getClosestParentOfType(upward: Boolean = true): T? {
```

Comment:
@tvanlaerhoven I could move this to some util file, any thoughts? or better keep it here?